packages/extension/src/store/jsonrpc.slice.ts

```diff
@@ -0,0 +1,31 @@
+import type { PayloadAction, Slice } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+
+export interface JsonRpcState {
+  network: {
+    providers: string[];
+    chainId: number;
+  };
+}
+
+const initialState: JsonRpcState = {
+  network: {
+    providers: ['https://api.mycryptoapi.com/eth'],
+    chainId: 1
+  }
+};
+
+const sliceName = 'jsonrpc';
+
+const slice: Slice<JsonRpcState> = createSlice({
```

Comment:
This slice looks to not be in use?

Reply:
It's not in use right now, but will be needed for handling requests to an actual JSON-RPC node.